.buildkite/steps/publish.sh

```diff
@@ -153,14 +153,16 @@ echo "--- Building and publishing stack"
 
 make setup build
 
-if [[ $BUILDKITE_BRANCH == "master" ]] ; then
+# Publish the top-level mappings only on version tagged releases
+if [[ "$BUILDKITE_TAG" =~ ^v ]] ; then
```

Comment:
If do a point release of an older version (say, by tagging `v1.1.2`), this will end up publishing it as the latest release? Is that expected?